ts/features/idpay/payment/screens/IDPayPaymentCodeInputScreen.tsx

```diff
@@ -50,40 +53,39 @@ const IDPayPaymentCodeInputScreen = () => {
           <VSpacer size={16} />
           <Body>{I18n.t("idpay.payment.manualInput.subtitle")}</Body>
           <VSpacer size={40} />
-          {/* FIXME replace with the new input field from the Design System 2.0  */}
-          <LabelledItem
-            isValid={isInputValid}
-            accessibilityLabel={I18n.t("idpay.payment.manualInput.input")}
-            inputMaskProps={{
-              type: "custom",
-              options: { mask: "SSSSSSSS" },
-              keyboardType: "default",
-              returnKeyType: "done",
-              value: inputState.value,
-              autoCapitalize: "none",
-              placeholder: I18n.t("idpay.payment.manualInput.input"),
-              onChangeText: value => {
-                setInputState({
+          <TextInput
+            textInputProps={{
+              inputMode: "",
+              autoCapitalize: "characters"
+            }}
```

Comment:
We should not allow auto correct here. See this: https://github.com/pagopa/io-app-design-system/pull/116

```suggestion
            textInputProps={{
              inputMode: "",
              autoCapitalize: "characters",
              autoCorrect: false
            }}
```

Reply:
It is now released: https://github.com/pagopa/io-app-design-system/releases/tag/v1.15.0

Reply:
@forrest57 we can now implement it: https://github.com/pagopa/io-app-design-system/releases/tag/v1.15.0

Reply:
addressed in [0476de1](https://github.com/pagopa/io-app/pull/5136/commits/0476de1c340e67a610b6f20a6098bdf07b8643f7) 👍